shared/js/background/https.es6.js

```diff
@@ -59,6 +60,20 @@ class HTTPS {
         return bloom
     }
 
+    /**
+     * Checks if provided hostname is local (e.g. ::1) or a private (e.g. 192.168.1.1)
+     * @param {string} host
+     * @returns {boolean}
+     */
+    isLocalOrPrivate (host) {
+        // if it looks like IP v6 remove [ and ] that wrap it ("[::1]" -> "::1") for ip library to work
+        if (host.startsWith('[') && host.endsWith(']')) {
+            host = host.slice(1, -1)
+        }
```

Comment:
TIL how IPv6 works in URLs. It's always in those brackets that, unfortunately, `ip` library does not understand. Thankfully it looks like domain name can't start with `[` so we can just strip it when we see it.